crates/install-wheel-rs/src/script.rs

```diff
@@ -16,6 +16,8 @@ pub struct Script {
     pub module: String,
     #[pyo3(get)]
     pub function: String,
+    #[pyo3(get)]
```

Comment:
Interestingly I forgot to add this at first (I assume it should be added) - but no test identified it.
Is there anyway to easily unit test this?